### PR TITLE
Feat alerts

### PR DESCRIPTION
**Pull request by Samarth-5109**
Screenshot (25)

Added Alerts of different types

### CODE REVIEW

**Comment by Samarth-5109** [comment]
I have hosted this branch on netlify but when I want to access the alerts page it shows "page not found"

**Review by Bharatrathod-81 — APPROVED** [review]
good work

File: components/alert/alerts.css [hunk 1]
```diff
@@ -0,0 +1,140 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;1,100&family=Shippori+Antique&display=swap');
+
+
+:root{
```
Comment (bharati-21) [inline components/alert/alerts.css]: If these colors are common to all components, a suggestion to add them to a common file.

File: components/alert/alerts.css [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    font-family: 'Roboto', sans-serif;
+}
+
+li{
+    list-style: none;
+}
+
+a{
+    text-decoration: none;
+}
```
Comment (bharati-21) [inline components/alert/alerts.css]: The same goes for resets as well. You could add them in a common file, say reset.css

File: components/alert/alerts.html [hunk 3]
```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
```
Comment (bharati-21) [inline components/alert/alerts.html:9]: You can add the author and description meta tags.

File: components/alert/alerts.css [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+    color: var(--secondary-color);
+}
+
+.github-icon{
+    color: var(--secondary-color);
+    font-size: 2rem;
+}
+
+/* SIDE-BAR */
+.wrapper{
```
Comment (bharati-21) [inline components/alert/alerts.css]: I see repeated styles (similar styles in docs.css). You can maybe combine them in a single CSS sheet.

**Review by bharati-21 — COMMENTED** [review]
Looks good to be merged. But you can add common CSS styles in one sheet and avoid repetition across multiple files in the next PR.

**Review by bharati-21 — APPROVED** [review]
Looks good to be merged!

**Comment by Samarth-5109** [comment]
Will be merging

**Pull request merged**